lib/matrix.gd

```diff
@@ -1677,6 +1677,8 @@ DeclareGlobalFunction( "RandomMat" );
 ##  returns a new random mutable <A>m</A><M>\times</M><A>m</A> matrix with integer
 ##  entries that is invertible over the integers.
 ##  Optionally, a random source <A>rs</A> can be supplied.
+##  If the option <A>domain</A> is given, random selection is made from <A>domain</A>, otherwise
+##  from <A>Integers</A>
 ##  <Example><![CDATA[
```

Comment:
[Suggestion] I think it will be useful to have an example demonstrating how this option is used, and also indicate that in the `Func` element above. This will also make the usage of `<A>` less confusing.

Reply:
I added an example. This will require two iterations as the manual check will discover the appropriate random values.

Reply:
First, there is a typo `domail`. Second, what about

```
<Func Name="RandomUnimodularMat" Arg='[rs ,] m [:domain:=d]'/>
...

##  If the option <C>domain</C> is given, random selection is made from the domain
##  <A>d</A> (allowing for larger coefficients), otherwise the default domain will be <Ref Var="Integers"/>.
```

Reply:
Good example, which uncovers my misunderstanding. I thought that a domain should be a domain in a GAP sense (especially since `Integers` is a domain). But that's not necessary: 
```
gap> IsDomain([-100..100]);
false
```
Maybe a different word/option name to avoid confusion? I guess `d` should be a collection for the code to work?

Reply:
The option is now called `random` for lack of a better word. (It is the set from which random numbers are chosen, matrix coeffs can be larger.